lms/product/canvas/_plugin/launch.py

```diff
@@ -0,0 +1,10 @@
+from lms.product.plugin.launch import LaunchPlugin
+
+
+class CanvasLaunchPlugin(LaunchPlugin):
+    def course_extra(self, lti_params):
```

Comment:
We could first super() and append this to the result but currently this is a canvas only thing, not sure how (if) the next one will look.